src/components/SelectNetworkAndAccount.svelte

```diff
@@ -1,38 +1,41 @@
 <script lang="ts">
   import { onMount } from 'svelte';
   import { allNetworks, type NetworkInfo } from '$lib/stores/networksStore';
-  import { getApi, updateConnectionStatus } from '$lib/polkadotApi';
+  import { createApi } from '$lib/polkadotApi';
   import { Account, fetchAccountsForNetwork } from '$lib/stores/accountsStore';
-  import { user } from '$lib/stores/userStore';
   import type { WsProvider, ApiPromise } from '@polkadot/api';
   import type { web3Enable, web3Accounts } from '@polkadot/extension-dapp';
   import DropDownMenu from '$components/DropDownMenu.svelte';
-  import { defaultDotApi } from '$lib/storeTypes';
+  import { user } from '$lib/stores/userStore';
+  import { formatNetwork, formatAccount, isValidURL } from '$lib/utils';
   import { dotApi } from '$lib/stores';
-  import { isValidURL, formatAccount, formatNetwork } from '$lib/utils';
 
-  export let selectedNetwork: NetworkInfo | undefined;
-  export let accountsStore: Map<string, Account>;
+  export let setNewValues: (account: Account) => void | undefined;
+  export let newAccount: Account | undefined;
+  export let accounts: Map<string, Account>;
```

Comment:
I thought you had changed this to `SS58Address` or similar?

Reply:
That was part of the split into a different PR. Now that the PR where we made that change is merged in, you'll see the update here. @rlaferla